devices/merrytek.js

```diff
@@ -0,0 +1,141 @@
+                2: 'minor_motion',
+                3: 'breathing',
+                4: 'abnormal_state',
+                5: 'initializing',
+                6: 'initialization_completed',
+            };
+            switch (dp) {
+            case tuya.dataPoints.mrytPresenceState:
+                return {
+                    occupancy: value > 0 ? true : false,
+                };
+            case tuya.dataPoints.mrytSensitivity:
+                return {
+                    sensitivity: value,
+                };
+            case tuya.dataPoints.mrytIlluminanceLux:
+                return {
+                    illuminance_lux: value,
+                };
+            case tuya.dataPoints.mrytIlluminanceCalibration:
+                return {
+                    illuminance_calibration: value,
+                };
+            case tuya.dataPoints.mrytKeepTime:
+                return {
+                    keep_time: value,
+                };
+            case tuya.dataPoints.mrytLED_Enable:
+                return {
+                    led_enable: value == 1 ? true : false,
+                };
+            case tuya.dataPoints.mrytMotionState:
+                return {
+                    states: lookup[value],
+                };
+            case 102:
+                return {
+                    LuxChangeTrigger: value,
+                };
+            case 106:
+                return {
+                    HoldTime: value,
+                };
+            case 108:
+                return {
+                    Scene: value,
+                };
+            case 110:
+                return {
+                    Minormotion: value,
+                };
+            case 111:
+                return {
+                    Breathing: value,
+                };
+            case 112:
+                return {
+                    Linght: value,
+                };
+            case 113:
+                return {
+                    Behavior: value,
+                };
+            default:
+                meta.logger.warn(`zigbee-herdsman-converters:tuya_smart_vibration_sensor: NOT RECOGNIZED ` +
+                        `DP #${dpValue.dp} with data ${JSON.stringify(dpValue)}`);
+            }
+        },
+    },
+};
+
+module.exports = [
+    {
+        fingerprint: [{modelID: 'TS0601', manufacturerName: '_TZE200_9xejegcy'}],
+        model: 'MSA201_Z',
+        vendor: 'Merrytek',
+        description: 'Large beam microwave occupancy sensor 24GHz',
+        fromZigbee: [fzLocal.merrytek_microwave_sensor],
+        toZigbee: [tzLocal.merrytek_microwave_sensor],
+        configure: async (device, coordinatorEndpoint, logger) => {
+            const endpoint = device.getEndpoint(1);
+            await reporting.bind(endpoint, coordinatorEndpoint, ['genBasic']);
+        },
+        exposes: [
+            e.occupancy(),
+            exposes.numeric('states', ea.STATE).withDescription('Motion state')
```

Comment:
If I understand correctly, this is an enum and not a number (e.g. `no_motion` from the lookup on line 42). If so plesae change this to `exposes.enum()`